sycl/source/detail/scheduler/commands.cpp

```diff
@@ -803,6 +805,12 @@ cl_int ExecCGCommand::enqueueImp() {
     NDRDescT &NDRDesc = ExecKernel->MNDRDesc;
 
     if (MQueue->is_host()) {
+      for (size_t I = 0; I < NDRDesc.Dims; ++I) {
+        if (NDRDesc.LocalSize[I] != 0 &&
+            NDRDesc.GlobalSize[I] % NDRDesc.LocalSize[I] != 0)
+          throw sycl::nd_range_error("Invalid local size for global size");
+      }
+
```

Comment:
Please, add comment here why you are checking this.

Reply:
This code is not actually needed: we already had proper checks for host device, but type of exception was incorrect: fixed it there and removed this code